src/it/scala/com/wavesplatform/it/IntegrationSuiteWithThreeAddresses.scala

```diff
@@ -28,6 +28,10 @@ trait IntegrationSuiteWithThreeAddresses extends FunSuite with BeforeAndAfterAll
   protected lazy val secondAddress: String = Await.result(sender.createAddress, 1.minutes)
   protected lazy val thirdAddress: String = Await.result(sender.createAddress, 1.minutes)
 
+  protected def getAccountEffectiveBalance(acc: String): Future[Long] = sender.effectiveBalance(acc).map(_.balance)
```

Comment:
In scala, `get` is usually not used: http://docs.scala-lang.org/style/naming-conventions.html
```
Scala does not follow the Java convention of prepending set/get to mutator and accessor methods (respectively). Instead, the following conventions are used:
For accessors of properties, the name of the method should be the name of the property.
```